scripts/python_scripts/generate_network_status.py

```diff
@@ -35,65 +36,55 @@
 
 
 def generate_networks_list():
+    sub_query = SubQueryDeploymentAPI(auth_token=token, org=organisation)
+    sub_query.collect_all_data()
+
     writer = MarkdownTableWriter(
         headers=["--", "Network", "Features", "Stage status",
                  "Prod status", "Stage commit", "Prod commit"],
-        value_matrix=generate_value_matrix(),
+        value_matrix=generate_value_matrix(sub_query),
         margin=1
     )
     writer.write_table()
+    asyncio.run(send_telegram_message(telegram_message))
     return writer
 
 
-def get_networks_list(folder):
-    onlyfiles = [f for f in listdir(folder) if isfile(join(folder, f))]
-    network_array = [name.split('.')[0] for name in onlyfiles if '.yaml' in name and 'project.yaml' not in name]
-    return network_array
-
-
-def get_deployments_list(network: str):
-    deployments = use_subquery_cli(subquery_cli_version, '--token', token, 'deployment',
-                                   'list', '--key', 'nova-wallet-'+network, '--org', 'nova-wallet', '-o', 'json')
-    production_instance = None
-    stage_instance = None
-
-    for instance in json.loads(deployments):
-        if (instance.get('type') == 'primary'):
-            if (production_instance): continue
-            production_instance = instance
-        else:
-            if (stage_instance): continue
-            stage_instance = instance
-
-    return production_instance, stage_instance
+def get_percentage(instance, network):
```

Comment:
I'd suggest to add [type hints](https://docs.python.org/3/library/typing.html) when possible to make it easier to understand which type do arguments have

Reply:
Improved type hints and added it through all functions, thx